data/scripts/eventcallbacks/player/default_onUpdateStorage.lua

```diff
@@ -0,0 +1,29 @@
+local lastQuestUpdate = {}
+
+local ec = EventCallback
+
+ec.onUpdateStorage = function(player, key, value, oldValue, isLogin)
+    if isLogin then
+        return
+    end
+
+    local playerId = player:getId()
+    local now = os.mtime()
+    if not lastQuestUpdate[playerId] then
+        lastQuestUpdate[playerId] = now
+    end
+
+    if now - lastQuestUpdate[playerId] <= 0 and Game.isQuestStorage(key, value, oldValue) then
```

Comment:
this never triggers, should be like this:
```suggestion
    if lastQuestUpdate[playerId] - now <= 0 and Game.isQuestStorage(key, value, oldValue) then
```